docs/topics/api/github.rst

```diff
@@ -11,9 +11,19 @@ This API provides an endpoint that works with GitHub to provide add-on validatio
 To set this up on a GitHub repository you will need to:
 
 * Go to `Settings > Webhooks & Services`
-* Add a new Webhook with Payload URL of `https://addons.mozilla.org/api/v3/github/`
+* Add a new Webhook with Payload URL of `https://addons.mozilla.org/api/v3/github/validate/`
+* Click `Send me everything`
 * Click `Update webhook`
 
+At this point the validator will be able to get the data, but won't be able to write a response to GitHub. To allow this:
```

Comment:
`To allow this:` is a weird sentence fragment. Can we write: `To enable responses to GitHub:`?